grails-app/conf/DataSource.groovy

```diff
@@ -1,8 +1,8 @@
 dataSource {
 	pooled = true
 	driverClassName = 'org.postgresql.Driver'
-	username = 'postgres'
-	password = 'postgres'
+	username = 'postres'
+	password = 'postres'

```

Comment:
changing this from `postgres` to `postres` will break the local environment for the other developers